datalad/interface/rerun.py

```diff
@@ -253,6 +253,12 @@ def _revs_as_results(dset, revs):
                 "Error on {}'s message: {}".format(rev, exc_str(exc)))
 
         if info is not None:
+            if len(parents) > 1:
```

Comment:
Just curious if you did git this situation in some  case? Eg if some settings manage to suck in the full run record into the merge message

Reply:
Cool, thanks